### ISSUE TITLE
Howell‐Jolly bodies

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4872]
**Preferred term label:**

Howell‐Jolly bodies

**Synonyms**
Presence of Howell-Jolly bodies

**Definition (free text, please give PubMed ID)**
Howell‐Jolly bodies are small, intra‐erythrocytic remnants of erythrocyte nuclei.
PMID: 26331020

**Parent term (use hpo.jax.org/app)**
Erythrocyte inclusion bodies HP:0020080

http://purl.obolibrary.org/obo/HP_0020080

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

PMID: 26331020
ORPHA:101351
asplenia, familial
asplenia, isolated congenital
congenital hypoplasia of spleen
hypoplasia of spleen
hyposplenia, isolated congenital
ICAS
splenic hypoplasia

**Your nano-attribution (ORCID)**
0000-0001-9969-8610

**Comment by pnrobinson** [comment]
adding as HP:0032550
